bridge/github/import_mediator.go

```diff
@@ -89,6 +89,7 @@ func NewImportMediator(ctx context.Context, client *rateLimitHandlerClient, owne
 	}
 	go func() {
 		mm.fillImportEvents(ctx)
+		ctx.Done()
```

Comment:
Hmm, I don't think `Done()` is supposed to be used like that. You normally need to wait on the returned channel within a `select`. Your logic make sense but `Done` doesn't actually terminate the context, it just allows you to wait that it is the case.

Reply:
Yes ... in this case it's simply used to wait and it could be wrapped in a select without an issue.  There's nothing special about `Done()` though it might more properly be written as `_ = ctx.Done()`.  I'm not married to this solution but figured since I had a working fix for this race, I'd at least start a discussion.

Reply:
I tried your branch and the race detector still trigger for me. I think that if you don't wrap `Done()` in a select, you are not actually waiting on anything. It gives you a channel that get ignored and the function terminate.
In any case, this problem is solved now. Thanks for finding it and looking at it :)

Reply:
Ha yes, that make sense with `<- ctx.Done()` :)